### PR TITLE
tools/image-info: bad syntax on rhel8 python

### PR DESCRIPTION
**Pull request by lavocatt**
We can't use:

with (a() as b, c() as d):
    pass

On older versions of python.

### CODE REVIEW

File: tools/image-info [hunk 1]
```diff
@@ -2569,133 +2569,133 @@ def partition_is_lvm(part: Dict) -> bool:
...
-                    key = parts[0]
-                    val = parts[1]
-
-                    # uid and gid must be integers
-                    if key == "uid" or key == "gid":
-                        val = int(val)
-
-                    options[key] = val
+    with tempfile.TemporaryDirectory() as mountpoint:
+          with host.ServiceManager(monitor=monitor.NullMonitor(1)) as mgr:
```
Comment (achilleas-k) [inline tools/image-info]: The indentation here is wrong

**Review by achilleas-k — CHANGES_REQUESTED** [review]

File: tools/image-info (hunk 1 above)
Comment (lavocatt) [reply tools/image-info]: I need to add a pep8 checker somewhere! Thanks.

**Comment by lavocatt** [comment]
On this PR https://github.com/osbuild/manifest-db/pull/78 I tested that when image-info is not changed on the PR the regression test immediately return. On this PR they should run because image-info was changed.

**Comment by lavocatt** [comment]
PR https://github.com/osbuild/manifest-db/pull/77 contains this commit and produced the update https://github.com/osbuild/manifest-db/pull/80 so everything's tested.

**Review by ondrejbudai — APPROVED** [review]

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**